fix(DIA-1249): follow-up tweaks for multiple artwork images in Discover Daily

iskounen opened the pull request:
This PR makes some follow-up tweaks to the work started in #11945.

1. Add an `artworkImageSwiped` tracking event when the image changes
2. Display a blurhash while images are loading
3. Decreased the tap areas from 33% of the screen to 20% on the left and right
4. Prevent the pagination bar from pushing the "save" button down

https://github.com/user-attachments/assets/f7a44a60-a18b-4e8a-80a0-065fdf6f45bc


### PR Checklist

- [x] I have tested my changes on the following platforms:
  - [x] **Android**.
  - [x] **iOS**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos** at least on **Android**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

### Changelog updates

<!-- 📝 Please fill out at least one of these sections. -->
<!-- ⓘ 'User-facing' changes will be published as release notes. -->
<!-- ⌫ Feel free to remove sections that don't apply. -->
<!-- • Write a markdown list or just a single paragraph, but stick to plain text. -->
<!-- 📖 eg. `Enable lotsByFollowedArtists` or `Fix phone input misalignment`. -->
<!-- 🤷‍♂️ Replace this entire block with the hashtag `#nochangelog` to avoid updating the changelog. -->
<!-- ⚠️ Prefix with `[NEEDS EXTERNAL QA]` if a change requires external QA -->

#### Cross-platform user-facing changes

- follow-up tweaks for multiple artwork images in Discover Daily

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

<!-- end_changelog_updates -->

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

ArtsyOpenSource commented:
<!--
  0 failure: 
  0 warning: 
  
  1 markdown notices
  DangerID: danger-id-Danger;
-->



### This PR contains the following changes:

- Cross-platform user-facing changes (follow-up tweaks for multiple artwork images in Discover Daily - iskounen)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d4ebf351615388fd6cdbdc631df2c35d1d8ba272
</p>

iskounen commented:
@araujobarret I added a ticket to follow-up with your suggestion: https://artsyproduct.atlassian.net/browse/DIA-1300

@damassi I added a ticket to follow-up on your suggestion as well. If someone can knock this out really quick, I think it's worth trying out: https://artsyproduct.atlassian.net/browse/DIA-1301
